generate-release/src/release_notes.rs

```diff
@@ -107,21 +112,23 @@ pub fn generate_release_notes(
 
 fn generate_metadata_block(
     title: &str,
-    authors: &[String],
+    author: &str,
+    contributors: &[String],
     pr_number: u64,
     file_name: &str,
 ) -> String {
     // TODO should probably add some weight for sorting
     format!(
         r#"[[release_notes]]
 title = "{title}"
-authors = [{authors}]
+authors = ["{author}",]
```

Comment:
Technically, this doesn't need to be an array anymore, but I'm not sure if it will break the generator tool if it's changed so we should probably just keep it like that. The trailing comma doesn't seem necessary though.

Reply:
It should still be an array since we will be manually adding back contributors to it depending on the level of contribution. And the release notes template then will still expect it to be an array, so it should be an array type.

As for the trailing comma, I left it there for ease of use when folks do add contributors back to the array on a case by case basis.